adapters/admatic/admatic_test.go

```diff
@@ -0,0 +1,21 @@
+package admatic
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v2/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAdmatic, config.Adapter{
+		Endpoint: "http://pbs.admatic.com.tr?host=admatic.rtb.admatic.com.tr"},

```

Comment:
should use same endpoint mentioned in bidder-info yaml i.e `"http://pbs.admatic.com.tr/?host={{.Host}}"`

this ensures url resolve functions works correctly for endpoint mentioned in bidder-info yaml